go.mod

```diff
@@ -22,10 +22,10 @@ require (
 	github.com/kr/pretty v0.3.0 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	github.com/rogpeppe/go-internal v1.9.0 // indirect
-	golang.org/x/net v0.12.0 // indirect
-	golang.org/x/sys v0.10.0 // indirect
-	golang.org/x/text v0.11.0 // indirect
-	google.golang.org/genproto v0.0.0-20230711160842-782d3b101e98 // indirect
+	golang.org/x/net v0.17.0 // indirect
```

Comment:
looks like dependabot covered this part: https://github.com/envoyproxy/go-control-plane/pull/795